ydb/docs/en/core/changelog-cli.md

```diff
@@ -2,13 +2,41 @@
 
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.8.0 {#2-8-0}
+
+Released on January 12, 2024. To update to version **2.8.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+**Features:**
+
+* Added configuration management commands for the cluster `ydb admin config` and `ydb admin volatile-config`.
```

Comment:
link to documentation?

Reply:
Translation of this documentation is not ready yet.